Detectors/TPC/reconstruction/include/TPCReconstruction/TrackTPC.h

```diff
@@ -0,0 +1,108 @@
+/// \file TrackTPC.h
+/// \brief Definition of the TPC Track
+/// \author Thomas Klemenz, TU Muenchen, thomas.klemenz@tum.de
+
+
+#ifndef ALICEO2_TPC_TRACKTPC
+#define ALICEO2_TPC_TRACKTPC
+
+#include "DetectorsBase/Track.h"
+
+#include "TPCSimulation/Cluster.h"
+
+#include <TClonesArray.h>
+
+namespace o2 {
+namespace TPC {
+
+/// \class TrackTPC
+/// This is the definition of the TPC Track Object
+
+
+class TrackTPC final : public o2::Base::Track::TrackParBase {
+  public:
+
```

Comment:
Thanks for the comment. In principle it would be good to start designing a class which can also be used later on already now. So I would prefer to choose a layout as close as possible to something final. 
For sure it will not be now, but we should learn from what we do now I think. Perhaps we can have a short chat.